### ISSUE TITLE
*.md not included in frameworks as resources

### ISSUE THREAD

**Issue by mikezucc** [pinterest/xchammer#80]
Not sure how this should be handled, but Readmes are not included in the final xcconfig for the libraries. They are a useful documentation source and should be visible within xcode.

**Comment by mikezucc** [comment]
Should this be done at the Bazel level or at XCHammer?

**Comment by jerrymarino** [comment]
Hey @mikezucc - XCHammer builds an Xcode project from a Bazel build graph - any files ( like ad-hoc documentation in a directory ) that aren't part of the build aren't part of the project. Is that what you're referring to?

**Comment by jerrymarino** [comment]
Do you have a suggestion about how to determine what ad-hoc files should be included? By using the build graph exclusively, projects are intentionally minimal. If we include tens-hundreds of thousands of files irrelevant to building, it will cause perf issues in XCHammer and Xcode - `README.md` is a subset of this.

Perhaps adopting *Github's* convention of `*README*` files  could be a good addition. Other than that, the recommendation is to gen another Xcode project in an Xcode workspace for docs, outside of XCHammer. Also, Jazzy is also widely used..

**Comment by mikezucc** [comment]
True, there wouldn't be an efficient filter for this unless a strict assumption was made. An example is our internal PINCore frameworks, where top level of each framework has a Readme.md. It would still be handy to CRUD these files from XCode. Identifying them is a problem. Perhaps this goes outside the scope of XCHammer and should be more a question of extending targets in Bazel files to define non-build critical source
